clair-scanner/scan.sh

```diff
@@ -70,13 +70,24 @@ function scan() {
 
 EXIT_STATUS=0
 
+for entry in "$DOCKER_TAR_DIR"/*.tar; do
+    [ -e "$entry" ] || continue
+    images=$(docker load -i "$entry" | sed -e 's/Loaded image: //g')
+    for image in $images; do
+        scan "$image"
+    done
+done
+
 if [ -n "<< parameters.image_file >>" ]; then
     images=$(cat "<< parameters.image_file >>")
     for image in $images; do
+        docker pull "$image"
         scan "$image"
     done
-else
-    scan "<< parameters.image >>"
+elif [ -n "<< parameters.image >>" ]; then
```

Comment:
With the new parameter, you can scan:
- one fixed image
- multiple images from a file
- multiple images from multiple tarfiles

But right now you can only use image_file OR image parameters. Can we allow using all the params at once?

Reply:
fixed in 3edee7f4e09747456b58a76d355466b4a0c9dc2d